### DISCUSSION TITLE
[SOLVED] adblock works a little and then fails

### DISCUSSION THREAD

**Discussion by labatts** [martinrotter/rssguard#763] [Q&A]
Hi, the adblock will work for a bit, and then eventually fail.  The node.js section in the settings appear to be okay -- there are checkmarks on all the sections at least.  This is on a flatpak installation on opensuse.  I have no idea if I am missing something or if this is a bug.  Thanks for any help.

**Comment by martinrotter** [comment]
Created ticket for this https://github.com/martinrotter/rssguard/issues/769, go there.
